tests/test_data/test_pipelines/test_augmentation.py

```diff
@@ -364,18 +364,40 @@ def test_color_jitter(self):
 
         results = copy.deepcopy(self.results)
         results['gt'] = (results['gt'] * 255).astype(np.uint8)
+        results['lq'] = [results['gt'], results['gt']]
 
-        target_keys = ['gt']
+        target_keys = ['gt', 'lq']
 
         color_jitter = ColorJitter(
-            keys=['gt'], brightness=0.5, contrast=0.5, saturation=0.5, hue=0.5)
+            keys=['gt', 'lq'],
+            brightness=0.5,
+            contrast=0.5,
+            saturation=0.5,
+            hue=0.5)
+        color_jitter_results = color_jitter(results)
+        color_jitter = ColorJitter(
+            keys=['gt', 'lq'],
+            channel_order='bgr',
+            brightness=0.5,
+            contrast=0.5,
+            saturation=0.5,
+            hue=0.5)
         color_jitter_results = color_jitter(results)
         assert self.check_keys_contain(color_jitter_results.keys(),
                                        target_keys)
         assert color_jitter_results['gt'].shape == self.img_gt.shape
 
         assert repr(color_jitter) == color_jitter.__class__.__name__ + (
-            f"(keys=['gt'], to_rgb=False)")
+            "(keys=['gt', 'lq'], channel_order=bgr)")
```

Comment:
`channel_order=bgr` should be `channel_order='bgr'`. 
Besides, its better to hardcode here, 
```
assert repr(color_jitter) == "ColorJitter(keys=['gt', 'lq'], channel_order='bgr')"
```